src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5260,6 +5260,12 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						profile->presence_hosts = switch_core_strdup(profile->pool, val);
 					} else if (!strcasecmp(var, "caller-id-type")) {
 						profile->cid_type = sofia_cid_name2type(val);
+					} else if (!strcasecmp(var, "force-caller-id-type-pai")) {
```

Comment:
Instead of enforcing only for PID, we can enhance this variable so that it can also enforce RPID if needed.

Reply:
@dev-ryanc should I handle to support for RPID and Preferred Identity or just rename the variable for now.

Reply:
It should allow us to enforce RPID or PID. In the future where a customer require RPID instead then we can just make configuration changes

Reply:
We're going to include these suggestions on v68.1

Reply:
task created : https://telnyx.atlassian.net/browse/ENGDESK-33618